From 9071e5a73e65f8032c0fbf283b1dbdfa67b0c617 Mon Sep 17 00:00:00 2001
From: Orit Wasserman <owasserm@redhat.com>
Date: Wed, 9 Oct 2013 10:09:15 +0200
Subject: [PATCH 20/25] pc: drop external DSDT loading

RH-Author: Orit Wasserman <owasserm@redhat.com>
Message-id: <1381313355-15641-11-git-send-email-owasserm@redhat.com>
Patchwork-id: 54806
O-Subject: [RHEL7.0 qemu-kvm v2 10/10] pc: drop external DSDT loading
Bugzilla: 921465
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Anthony Liguori <aliguori@us.ibm.com>

This breaks migration and is unneeded with modern SeaBIOS.

Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
Message-id: 1376316104-11269-1-git-send-email-aliguori@us.ibm.com
(cherry picked from commit 9fb7aaaf4c58c9108327f0ae4766087e3e496b47)
---
 hw/i386/pc_piix.c | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 06e9eb8..af1f602 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -96,7 +96,6 @@ static void pc_init1(MemoryRegion *system_memory,
                               OBJECT(icc_bridge), NULL);
 
     pc_cpus_init(cpu_model, icc_bridge);
-    pc_acpi_init("acpi-dsdt.aml");
 
     if (kvmclock_enabled) {
         kvmclock_create();
-- 
1.7.1

